_posts/2015-08-07-symbolic-operators.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: Symbolic operators and type classes for Cats
+
+meta:
+  nav: blog
+  author: non
+  pygments: true
+---
+
+Symbolic operators and type classes for Cats
+============================================
+
+This post is an introduction into how operators are implemented in Cats and has been originally published in [August 2015](https://gist.github.com/non/3abdb35a72c39276d3d9).
+Some more details can be found in the [previous post]({% post_url 2015-08-06-machinist %}).
+
+One of the simplest and most recognizable type classes is the semigroup.
+This type class abstracts over the ability to combine values of a certain
+type in an associative manner.
+
+<div class="side-note">
+  What does <em>associativity</em> mean?
+  We call an operation `⊕` associative, if for all `a`, `b` and `c`, `a ⊕ (b ⊕ c) = (a ⊕ b) ⊕ c` holds.
```

Comment:
[Even backticks do not work in side-note.](https://github.com/typelevel/typelevel.github.com/pull/47#issuecomment-124268180)

Also, can you see those symbols @tpolecat?